### PR TITLE
Update cached package list

### LINKED ISSUE #5: Update the cdnjs data
The cdnjs data has not been updated for over a year. Please update to a more recent data source.

### PR DESCRIPTION
**Pull request by goober**
Fixes #5

### CODE REVIEW

**Comment by passy** [comment]
Sorry, nit but since the diff is pretty much uninspectable, could you rephrase the commit message to state that it's the cdnjs package cache that you update?

**Comment by goober** [comment]
@passy fixed.

**Comment by passy** [comment]
Thank you! :sparkles:
